src/main/java/frc/robot/subsystems/CDSSubsystem.java

```diff
@@ -54,6 +60,42 @@ public void stopCDS() {
     CDSWheelControllerOne.getSparkMax().set(0.0);
     CDSBeltController.getSparkMax().set(0.0);
     SmartDashboard.putNumber("CDS Belt Speed", 0.0);
+  }
+  
+  public int[] getSensorStatus() {
+    int frontStatus = colorSensorOne.getProximity() > 800 ? 1: 0;
+    int backStatus = backBeamBreak.get() ? 1: 0;
+    int[] beamBreakArray = {frontStatus, backStatus};
```

Comment:
Why not return a boolean array?

Reply:
The way we calculate ball count is to get the number of sensors that are currently being covered by a ball. If more than 2 sensors are covered, then we know we have too many. Definitely possible there's a better way to do this.